### ISSUE TITLE
null values are being treated as zeros in LineChart

### ISSUE THREAD

**Issue by scottdickerson** [carbon-design-system/carbon-charts#463]
#### I am submitting a...

- [x] Feature request
- [ ] Design defect
- [ ] Source code defect
- [ ] Demo/documentation defect
- [ ] Other

#### charts version:
0.16.25
### Issue description
If I pass null values to a LineChart, instead of getting gaps in the data, the graph graphs the value as a 0 on the y-axis.

#### Steps to produce the issue
```
{
					date: new Date(2019, 0, 1),
					value: 10000
				},
				{
					date: new Date(2019, 0, 1),
					value: null
				},
```

#### Current behavior
Graphs as zeros

#### Expected behavior
I'm expecting something a little like this: 
https://www.carbondesignsystem.com/data-visualization/axes-and-labels/#gaps-in-data

### Screenshot or recording
![image](https://user-images.githubusercontent.com/6663002/71983707-f9ba4b00-31ec-11ea-9977-467b07b3f810.png)

**Comment by scottdickerson** [comment]
@theiliad just an open question on this one, for a multi-line graph, would you only render the 'gap' in data if all of the values of a line were null for a x data point?

If one of the lines values was null for a timestamp and the rest of the lines actually had values, how would it render?

**Comment by theiliad** [comment]
Just from a mathematical perspective I’d say if a point between 2 other
points is undefined, then there’s no real domain between them, hence you’d
get the gap to show.

**Comment by anne-stevens** [comment]
Nulls should not be rendered as zeros.  A zero is an actual value that a user, sensor or system acutally input.  A null the absence of a value.  It is conceptually very different than zero.  In the line chart example, nulls should simply not be plotted.

**Comment by gvnmagni** [comment]
We support what Anne is suggesting and, visually, we would remove any line between datapoints if there are missing data between them.

Technically, this issue https://github.com/carbon-design-system/carbon-charts/pull/464 shows the solution to our problem, it has already been implemented.

**Comment by gvnmagni** [comment]
Russel, this image show the solution that was already developed for a different issue and that solves this one as well.

![72170731-65dcb080-339f-11ea-88a1-e1b91cb33c5e](https://user-images.githubusercontent.com/18484576/76977599-d0791200-6935-11ea-9329-7e242ef49048.png)

**Comment by theiliad** [comment]
Closed by #464
